store-js/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "short_name": "Store js",
```

Comment:
Is is the name the app will get on the phone? if so, would be good to include `Waku` in it.

Reply:
Addressed it here #182